### DISCUSSION TITLE
Recommended weapons for reserves

### DISCUSSION THREAD

**Discussion by janstehno** [janstehno/cotw-companion#39] [Ideas]
As title says, for each reserves, the "best" weapons would be shown (like callers). I would probably use the same method I am using for each animal's recommended weapons.

[original discussion](https://github.com/janstehno/cotwcompanion/discussions/34)

**Comment by JeanRoxus** [comment]
It could be located within the reserves section, under fauna and callers

**Reply by janstehno** [reply]
Yeah, that seems to be the best location for this feature.

**Comment by janstehno** [comment]
What should I show? Should I show only the best rifles or should I make 4 lists again of the best of each weapon?

**Reply by JeanRoxus** [reply]
I'm not so clear on the idea anymore, but as you said, it seems good to me to place only the best rifles that specifically cover all the classes of that certain reserve and if the maximum weight of the inventory (20-23 kg) could be taken into account, it would still be better.

 The best combination of callers and weapons to reserve as a build for each location.

**Reply by janstehno** [reply]
I see. There would eventually be a problem with rifles only because there would be just 1 shotgun, 1 bow or 1 crossbow, and 2/3 same handguns. The rest to be solved would be the rifles. I could do separate lists for each type of weapon. I definitely can't see a solution for a list that would combine all the weapon types.

**Reply by Caldas** [reply]
Don't try to tackle all at once.

Focos the weapon recommendation for the best fit for each gun type and what the percentage that fauna it's covers

**Comment by janstehno** [comment]
So I have experimented a little with this feature. However after some time I am really thinking about its usefulness.

I found out that I have limited options in terms of comparing weapons to determine the best possible loadout for each reserve. The easiest criteria I can do are:

- penetration
- effective range
- number of overlapping classes
- prefer non-DLC weapons

Which makes sense, the better penetration the easier is to hit a vital organ, the better effective range the easier it is to shoot from further distance. Overlapping classes is a no-brainer as you want to carry as few weapons as possible. Preffering non-DLC weapons is something I think is better overall.

From what I tried and the results I got I found out that the weapons are pretty much the same for every reserve. Here are two examples:

### Hirschfelden Hunting Reserve - classes 1-6 and 9, with DLC weapons **disabled**
<img src="https://github.com/janstehno/cotw-companion/assets/72458808/d26d6aa9-6e6e-46df-a9be-088a9c946be1" width="200">
<br><br>

### Hirschfelden Hunting Reserve - classes 1-6 and 9, with DLC weapons **enabled**
<img src="https://github.com/janstehno/cotw-companion/assets/72458808/65c472d4-6ef5-4c5f-ac60-487c9caaeab5" width="200">
<br><br>

### Te Awaroa National Park - classes 1-6, with DLC weapons **disabled**
<img src="https://github.com/janstehno/cotw-companion/assets/72458808/65bf27c3-4d63-4f53-a083-a8fa3f7cc012" width="200">
<br><br>

### Te Awaroa National Park - classes 1-6, with DLC weapons **enabled**
<img src="https://github.com/janstehno/cotw-companion/assets/72458808/99961c37-51a1-4b11-9678-78dc58606fe6" width="200">
<br><br>

From what you can see, the weapons will be always identical, the only difference being the rifle for classes 7-9 and the handgun for classes 4-9 are not included for Te Awaroa National Park.

So what I am wondering is if this feature is even needed when the list will always be the same most of the time. Let me know.

**Comment by janstehno** [comment]
Added in the version 1.10.3
